### PR TITLE
REF: misplaced arithmetic tests

### PR DESCRIPTION
**Pull request by jbrockmendel**

### CODE REVIEW

File: pandas/tests/series/test_arithmetic.py [hunk 1]
```diff
@@ -64,6 +66,65 @@ def _constructor(self):
...
 
+    def test_flex_add_scalar_fill_value(self):
+        # GH12723
+        s = Series([0, 1, np.nan, 3, 4, 5])
+
+        exp = s.fillna(0).add(2)
+        res = s.add(2, fill_value=0)
+        tm.assert_series_equal(res, exp)
+
+    pairings = [(Series.div, operator.truediv, 1), (Series.rdiv, ops.rtruediv, 1)]
```
Comment (jreback) [inline pandas/tests/series/test_arithmetic.py]: can you paramaterize

File: pandas/tests/series/test_arithmetic.py [hunk 2]
```diff
@@ -131,6 +286,50 @@ def test_comparison_flex_basic(self):
             with pytest.raises(ValueError, match=msg):
                 getattr(left, op)(right, axis=1)
 
+    def test_comparison_flex_alignment(self):
```
Comment (jreback) [inline pandas/tests/series/test_arithmetic.py]: I thought you were splitting out arithemtic & comparisons elsewhere? seems reasonable to do here as well.

**Review by jreback — CHANGES_REQUESTED** [review]

File: pandas/tests/series/test_arithmetic.py (hunk 1 above)
Comment (jbrockmendel) [reply pandas/tests/series/test_arithmetic.py]: this is constructing the parameters that go into parametrize on L88

File: pandas/tests/series/test_arithmetic.py (hunk 2 above)
Comment (jbrockmendel) [reply pandas/tests/series/test_arithmetic.py]: this is in the class `TestSeriesFlexComparison`

**Review by jreback — APPROVED** [review]

**Comment by jreback** [comment]
thanks

**Pull request merged**